affiliated/registry.json

```
@@ -1,5 +1,23 @@
 {
     "packages": [
+        {
+            "name": "ZodiPy",
+            "maintainer": "Metin San <metin.san@astro.uio.no>",
+            "stable": true,
+            "home_url": "https://cosmoglobe.github.io/zodipy/",
+            "repo_url": "https://github.com/cosmoglobe/zodipy",
+            "pypi_name": "zodipy",
+            "description": "ZodiPy is a Python tool for simulating the zodiacal emission that a solar system observer sees, either in the form of timestreams or binned HEALPix maps.",
```

```suggestion
            "description": "ZodiPy is a Python tool for simulating the zodiacal emission that a solar system observer sees, either in the form of timestreams or binned HEALPix maps.",
            "coordinated": false,
```
This is a required field that we have forgotten to add to the template yet.